src/shared/dropdown-list/types.ts

```diff
@@ -15,6 +15,7 @@ export interface DropdownDisplayProps<T, V> {
         item: T,
         args: ListItemRenderArgs
     ) => JSX.Element | undefined;
+    secondaryLabelExtractor?: (item: T) => string | undefined;
```

Comment:
I would prefer if we add on to the `listExtractor` instead. To be something like that
```
listExtractor?: ((item: T) => string | ListItemDisplayProps) | undefined;
```
and 

```
interface ListItemDisplayProps {
    title: string;
    secondaryLabel?: string | undefined
}
```

Reply:
then in our Dropdown list we'll check, if it is a string, it means the user just wants title. Then we'll render the string into the title. But if it is an object, we'll render the title and if secondaryLabel is specified, we render it as well. 

Anyway I realised those functions typings is lacking a parenthesis. Might need you to help me add for this file. Because e.g.
`valueExtractor?: (item: T) => V | undefined;`
it actually meant can pass in a function or pass undefined. But if specify like that, it means, must pass function that has return type of either `V` or `undefined`

So it should be 
`valueExtractor?: ((item: T) => V) | undefined;`